cartoframes/datasets.py

```diff
@@ -334,6 +344,8 @@ def clean_dataframe_from_carto(df, table_columns, decode_geom=False):
     for column_name in table_columns:
         if table_columns[column_name]['type'] == 'date':
             df[column_name] = pd.to_datetime(df[column_name], errors='ignore')
+        elif table_columns[column_name]['type'] == 'boolean':
```

Comment:
Good catch, I wasn't aware that bools weren't properly converted. Maybe the second bullet item in the docstring should be expanded to include `process date and boolean columns`